src/Elmish.WPF/Binding.fs

```diff
@@ -73,34 +73,34 @@ module internal BindingLogic =
 
             update curTarget nextSource (curTargetIdx + 1)
             update nextTarget curSource curTargetIdx
-            
```

Comment:
Crap sorry, I forgot to disable "trailing whitespace visualizer", let me revert the unwanted changes.